avidemux_ru.ts

```diff
@@ -8417,7 +8418,7 @@ Start and/or end time would be out of range</source>
     <message>
         <location line="-3"/>
         <source>Main10</source>
-        <translation type="unfinished"></translation>
+        <translation>Основной10</translation>
```

Comment:
If you feel that the profile name needs translation, I would suggest writing the profile name with a space. "Main10" in the source is apparently a mixup of "Main 10" and "main10" (the command line parameter) and needs to be fixed.

Reply:
Okey, I'll fix it